optuna/multi_objective/_hypervolume/wfg.py

```diff
@@ -18,7 +18,7 @@ def __init__(self) -> None:
         self._reference_point = None
 
     def _compute(self, solution_set: np.ndarray, reference_point: np.ndarray) -> float:
-        self._reference_point = reference_point
+        self._reference_point = reference_point.copy()
```

Comment:
This fixes an unintended side effect that _compute() overwrites the original user-defined reference_point at https://github.com/optuna/optuna/blob/master/optuna/multi_objective/_hypervolume/utils.py#L43 .

Reply:
Thank you for the good catch! This fix can be done in another PR. How about making a new PR to fix this?

Reply:
Thank you.
I've just made another PR for this (https://github.com/optuna/optuna/pull/1812).